CONTRIBUTING.md

```diff
@@ -173,7 +173,7 @@ Please ensure that all React UI components contributed meet the following guidel
 
   ```js
   # Badge/Badge.js
-  export { Badge as default } from 'react-bootstrap';
+  export { Badge as default } from 'react-bootstrap/lib/Badge';
```

Comment:
This is the commonjs version.  While that is needed for tests it will be better to use the ES version.  We are doing something similar in @patternfly/react-core for using babel to import either lib or es based off the module type.  We are actually using this plugin: https://www.npmjs.com/package/babel-plugin-transform-imports which imcludes an example for `react-bootstrap.`